app/gui/src/presenter/project.rs

```diff
@@ -9,6 +9,7 @@ use crate::executor::global::spawn_stream_handler;
 use crate::presenter::graph::ViewNodeId;
 use enso_frp as frp;
 use ide_view as view;
+use view::project::WayOfOpeningSearcher;
```

Comment:
Please fix import ordering.

Reply:
Ordering is actually matching our style guide, I added a few new lines here and there